contrib/completion/bash/docker

```diff
@@ -529,6 +529,14 @@ __docker_complete_log_driver_options() {
 			COMPREPLY=( $( compgen -W "false true" -- "${cur##*=}" ) )
 			return
 			;;
+		splunk-verifyconnection)
```

Comment:
please merge this into the preceding case because both share the same completion:

    splunk-insecureskipverify|splunk-verifyconnection)